hasher-matcher-actioner/webapp/src/pages/Dashboard.jsx

```diff
@@ -69,42 +69,31 @@ function toUFlotFormat(graphData) {
   return [timestamps, values];
```

Comment:
These changes were just ordering. Dont use a function before you define it. Tested by looking at Dashboard